shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -172,9 +172,7 @@ - (void)updateVisualEffectView:(UIVisualEffectView*)visualEffectView {
   visualEffectSubview.layer.backgroundColor = UIColor.clearColor.CGColor;
   visualEffectView.frame = _frame;
 
-  if (_backdropFilterView != visualEffectView) {
-    _backdropFilterView = [visualEffectView retain];
-  }
+  self.backdropFilterView = visualEffectView;
```

Comment:
The old `_backdropFilterView` value wasn't released when being set to a new visualEffectView.  Instead of adding a release, instead use the existing `retain` property setter.